.github/actions/spelling/expect/expect.txt

```diff
@@ -508,6 +508,7 @@ dealloc
 Debian
 debolden
 debugtype
+DECAC
```

Comment:
Note that this was meant to be `DECAC1`, but the spelling bot doesn't allow digits here. I'm not sure if there's a better way to handle this.

Reply:
I'm ok with this.